### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by DevelopnTest**
1st commit

### CODE REVIEW

File: LetterGradebyScoreAndProject/src/LetterGrade.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+//	        System.out.println("The grade of is " + grade);	        
+		} catch (NumberFormatException ex) {
+			System.out.println("Not an integer !");
+		} catch (IOException e) {
+			e.printStackTrace();
+		}     
+
+	}  
+
+	public char letterGrade(int exam, int project) {
```
Comment (cxl4252) [inline LetterGradebyScoreAndProject/src/LetterGrade.java]: should provide comment for this method

File: LetterGradebyScoreAndProject/src/LetterGradeTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	}
+
+	@Test
+	public void testInvalidInput() {
+//		assertEquals(lgrade.letterGrade('a', 101),'X');	
+		assertEquals(lgrade.letterGrade(101,'a'),'X');	
+	}
+	
+	@Test 
+	public void testConsoleOutput() {
```
Comment (cxl4252) [inline LetterGradebyScoreAndProject/src/LetterGradeTest.java]: **DevelopnTest**

> it seems to be an empty method

**Review by cxl4252 — COMMENTED** [review]
Overall, the code is good, just need to add comments and remove an empty test method

File: LetterGradebyScoreAndProject/src/LetterGrade.java (hunk 1 above)
Comment (DevelopnTest) [reply LetterGradebyScoreAndProject/src/LetterGrade.java]: OK

File: LetterGradebyScoreAndProject/src/LetterGradeTest.java (hunk 2 above)
Comment (DevelopnTest) [reply LetterGradebyScoreAndProject/src/LetterGradeTest.java]: I will remove it.

Comment (cxl4252) [inline LetterGradebyScoreAndProject/src/LetterGradeTest.java]: Sorry I delete your reply accidentally

Comment (cxl4252) [reply LetterGradebyScoreAndProject/src/LetterGradeTest.java]: I will remove it.

Comment (cxl4252) [reply LetterGradebyScoreAndProject/src/LetterGradeTest.java]: Sorry I delete your reply accidentally

File: LetterGradebyScoreAndProject/src/LetterGrade.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+		double score;
+		char grade;
+		score = exam*0.7 + project*0.3;
```
Comment (cxl1521) [inline LetterGradebyScoreAndProject/src/LetterGrade.java]: the 0.7 and 0.3 are magic number smells, they shall be avoid...
try the following code
`final double EXAM_SCORE_PERCENT = 0.7;`
`final double PROJECT_SCORE_PERCENT = 0.3`
`score = exam* EXAM_SCORE_PERCENT + project* PROJECT_SCORE_PERCENT;`

File: LetterGradebyScoreAndProject/src/LetterGradeTest.java (hunk 2 above)
Comment (cxl1521) [reply LetterGradebyScoreAndProject/src/LetterGradeTest.java]: it is OK ...ha ha

**Review by cxl1521 — CHANGES_REQUESTED** [review]
the magic number smells shall be refactored before you commit and merge the code

File: LetterGradebyScoreAndProject/src/LetterGrade.java (hunk 3 above)
Comment (DevelopnTest) [reply LetterGradebyScoreAndProject/src/LetterGrade.java]: Sure, will do it soon

Comment (DevelopnTest) [reply LetterGradebyScoreAndProject/src/LetterGrade.java]: Sure, I will do it soon...Thanks for the comments ^^

File: LetterGradebyScoreAndProject/src/LetterGrade.java (hunk 1 above)
Comment (DevelopnTest) [reply LetterGradebyScoreAndProject/src/LetterGrade.java]: @cxl1521 
The method comment has been added and the used test method was removed.

File: LetterGradebyScoreAndProject/src/LetterGrade.java [hunk 4]
```diff
@@ -32,10 +34,14 @@ public static void main(String[] args) {
 
 	}  
 
+	/*
+	 * This method is to calculate the letter grade with two inputs:
+	 * exam score and project score
+	 */
 	public char letterGrade(int exam, int project) {
```
Comment (DevelopnTest) [inline LetterGradebyScoreAndProject/src/LetterGrade.java]: @cxl4252 the magic number smells have been removed. Thanks.

**Review by cxl4252 — COMMENTED** [review]
The code looks fine. I have further comment. You can merge the code if @cxl1521  has no problem with the code.

**Review by cxl1521 — APPROVED** [review]
The code looks perfect. Good job!!!

**Pull request merged**